gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -571,6 +573,16 @@
 
   private static final String PEGASUS_PLUGIN_CONFIGURATION = "pegasusPlugin";
 
+  // Enable the use of generic pegasus schema compatibility checker
+  private static final String ENABLE_PEGASUS_SCHEMA_COMPATIBILITY_CHECK = "pegasusPlugin.enablePegasusSchemaCompatibilityCheck";
```

Comment:
"pegasusPlugin." prefix is used to make the configuration name unique, just in case the configuration name is duplicate with other configuration name in other plugins. It is the same format as other pegasus configurations. I will keep this part as is.